src/lib/metric-data-getters/elasticsearch.test.ts

```diff
@@ -0,0 +1,62 @@
+import moment from 'moment';
+
+import { ElasticsearchMetricDataGetter } from './elasticsearch';
+
+describe('Elasticsearch', () => {
+  describe('getData', () => {
+    it('should get data from prometheus', async () => {
+      const getSeries = jest.fn();
+
+      const rangeStart = moment();
+      const rangeStop = rangeStart.clone().add(1, 'day');
+      const aValue = 123456.789;
+
+      getSeries.mockReturnValue(Promise.resolve([
+        { label: 'instance', metrics: [{ date: rangeStart.toDate(), value: aValue }] },
+      ]));
+
+      const dg = new ElasticsearchMetricDataGetter({ getSeries } as any);
+
+      const data = await dg.getData(
+        ['loadAvg'],
+        'abc-def',
+        moment.duration(1, 'minute'), rangeStart, rangeStop,
+      );
+
+      expect(data).toHaveProperty('loadAvg');
+      expect(data.loadAvg).not.toBeUndefined();
+      expect(data.loadAvg.length).toEqual(1);
+      expect(data.loadAvg[0].label).toEqual('instance');
+      expect(data.loadAvg[0].metrics).toContainEqual({ date: rangeStart.toDate(), value: aValue });
+    });
+
+    it('should get data filter out the results of bad queries', async () => {
```

Comment:
Is this trying to say "*get data should filter out the results of bad queries*"?